libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/DrillTargetType/DrillTargetTypeListItem.tsx

```diff
@@ -25,12 +30,23 @@ const DrillTargetTypeListItem: React.FC<IDrillTargetTypeListItemProps> = ({
     });
 
     return (
-        <SingleSelectListItem
-            className={className}
-            title={item.title}
-            isSelected={isSelected}
-            onClick={handleClick}
-        />
+        <BubbleHoverTrigger hideDelay={0} showDelay={0}>
+            <SingleSelectListItem
+                className={className}
+                title={item.title}
+                isSelected={isSelected}
+                onClick={handleClick}
+            />
+            {item.disabled && item.disableTooltipMessage ? (
+                <Bubble
+                    className="bubble-primary"
+                    alignPoints={[{ align: "cr cl" }]}
+                    arrowOffsets={{ "cr cl": [15, 0] }}
```

Comment:
Please extract these two objects to constant outside of the component.

Reply:
I have updated this. Thank you!